### PR TITLE
Adds support for `table` renderer using `ember-frost-table`

### PR DESCRIPTION
**Pull request by cstolli**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #patch# - backwards-compatible bug fix
- [x] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG

Please add a description of your change here, it will be automatically prepended to the `CHANGELOG.md` file.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -9,5 +9,8 @@
     "showdown": "1.5.1",
     "sinonjs": "1.17.1",
     "clockpicker-seconds": "~0.1.6"
+  },
+  "resolutions": {
+    "route-recognizer": "^0.2.3"
```
Comment (sandersky) [inline bower.json]: Where did this come from?

File: tests/dummy/app/pods/view/renderers/values/index.js [hunk 2]
```diff
@@ -28,6 +28,13 @@ export default {
   'property-chooser': {},
   select: {},
   string: {},
+  table: {
+    things: [
+      {foo: 'hello', bar: 'there', bazz: 'buddy'},
```
Comment (sandersky) [inline tests/dummy/app/pods/view/renderers/values/index.js]: I'm digging the South Park reference.

File: bower.json (hunk 1 above)
Comment (cstolli) [inline bower.json]: idk, I did a bower install and it asked me to resolve a version conflict.  I did a `!` to put it in the `bower.json`

**Comment by sandersky** [comment]
:+1:

**Pull request merged**